engine/orchestration/src/com/cloud/agent/manager/ClusteredAgentManagerImpl.java

```diff
@@ -1411,14 +1411,12 @@ protected void runInContext() {
                         final double managedHostsCount = allManagedRoutingAgents.size();
                         if (allHostsCount > 0.0) {
                             final double load = managedHostsCount / allHostsCount;
-                            if (load >= ConnectedAgentThreshold.value()) {
-                                s_logger.debug("Scheduling agent rebalancing task as the average agent load " + load + " is more than the threshold " +
-                                        ConnectedAgentThreshold.value());
+                            if (load > ConnectedAgentThreshold.value()) {
+                                s_logger.debug("Scheduling agent rebalancing task as the average agent load " + load + " is more than the threshold " + ConnectedAgentThreshold.value());
                                 scheduleRebalanceAgents();
                                 _agentLbHappened = true;
                             } else {
-                                s_logger.debug("Not scheduling agent rebalancing task as the averages load " + load + " is less than the threshold " +
-                                        ConnectedAgentThreshold.value());
+                                s_logger.debug("Not scheduling agent rebalancing task as the average load " + load + " has not crossed the threshold " + ConnectedAgentThreshold.value());
```

Comment:
I can see teh '>=' -> '>' change in the line above. Has this line any logical changes as well?